### ISSUE TITLE
Unexpected type returned from getState()

### ISSUE THREAD

**Issue by Svenlaa** [paynl/sdk#172]
## The issue
The `getState()` method at line 36 in [src/Result/Transaction/Status.php](https://github.com/paynl/sdk/blob/1bab7054213a0aae43221a67d7082d179ef4282e/src/Result/Transaction/Status.php) indicates a return type of integer.
**Instead, it returns a string.** 

## More info
The underlying api call returns a string. That is expected. https://docs.pay.nl/developers#transaction-status

## Potential fix
My issue could be fixed by either changing the expected return type, or parsing it as `int` before returning it.

## How to reproduce
1. call Transaction::status()
2. call getState() on the returned value
3. you get a string

### (Minimal) Code example
```
$payTransaction = \Paynl\Transaction::status($transactionId);
$payTransaction->getState(); // "100"
```

**Comment by woutse** [comment]
Hi, thanks for your message. You have a point: ) We'll replace the description with "mixed" for now.

**Comment by woutse** [comment]
Thanks again for your message. Fixed in 2.2.2
